tools/internal_ci/linux/grpc_reverse_test_python.sh

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 The gRPC Authors
```

Comment:
Why is this called  "reverse_test"? I don't know what it means and it's likely that other won't know either.  

Looks like this test builds grpc python packages, installs them and then tries to run firestore unit tests?

Questions:
- do we really need to run this on PRs? That means extra test load and latency. It seems that once we get the test green, we can only run it on master to make sure we haven't broken firestore (but it seems we can only break them by pushing a bad release)

- look like this is basically a kind of a distrib test - we build python package and then try to use it (with firestore in this case). So perhaps we should just add this test as a python distrib test (by adding the job to distribtest_targets.py) or at least classify it like one (Having many uncategorized test jobs makes it hard to monitor test health and to maintain stuff, because everything is "custom").

Reply:
> do we really need to run this on PRs?

You are right. We don't need to run this on PRs. So, I updated it internally to twice a day.

> look like this is basically a kind of a distrib test

I wanted to run it as distrib test, and I found distrib tests focus on testing platform compatibility. There are two blockers: 1) we need to install all Python version in all platform (currently, we only test 2.7); 2) running this dependency test is time consuming (might slow down our release process).